parsers/http/actions/return.go

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright 2019 HAProxy Technologies
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//nolint:dupl
+package actions
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/haproxytech/config-parser/v3/common"
+)
+
+type Return struct { // http-request return [status <code>] [content-type <type>] [ {default-errorfile | <content-format> <content>} ] [ hdr <name> <fmt> ]* [{if | unless} <condition>]
+	Comment       string
+	Status        *int64
+	ContentType   string
+	ContentFormat string
+	Content       string
+	Hdrs          []*Hdr
+	Cond          string
+	CondTest      string
+}
+
+type Hdr struct {
+	Name string
+	Fmt  string
+}
+
+func IsPayload(in string) bool {
+	_, ok := map[string]struct{}{
+		"string":    {},
+		"lf-string": {},
+		"file":      {},
+		"lf-file":   {},
```

Comment:
creating this map every time that `http-request return` gets parsed will
put a lot of pressure on GC if many `http-request return ...` lines exists.

I think its better to have this a global (unexported/private) const

Reply:
done